File: geek-university/algoritmo-e-logica-de-programacao/java/exercicio_9.java
import java.util.Scanner;

public class exercicio_9 {

	public static void main(String[] args) {
		int metros, centimetros;
		Scanner teclado = new Scanner(System.in);

		System.out.print("Informe o valor em metros: ");
		metros = teclado.nextInt();

		centimetros = (metros * 100);

		System.out.println(metros + " metros em centimetros eh " + centimetros);
		
		teclado.close();
	}

}

File: geek-university/algoritmo-e-logica-de-programacao/c/exercicio_9.c
#include <stdio.h>

int main() {
	int metros, centimetros;

	printf("Informe o valor em metros: ");
	scanf("%d", &metros);

	centimetros = (metros * 100);

	printf("%d em centimetros eh igual a %d", metros, centimetros);
}
